apps/klasses/models.py

```diff
@@ -81,3 +83,27 @@ def save(self, *args, **kwargs):
             self.created = timezone.now()
         self.modified = timezone.now()
         return super().save(*args, **kwargs)
+
+    def homeworks_completely_graded(self):
```

Comment:
If this is on homework model, can probably pull the first part of this function and rename it:
`completely_graded` or some such?

Reply:
This is actually on the klass, so we know if all of the submissions on each homework have been graded.